src/main/java/com/saucedemo/pages/Login.java

```diff
@@ -7,6 +7,7 @@
 
 public class Login {
     private WebDriver driver;
+    private String loginUrl = "https://www.saucedemo.com/";
```

Comment:
so since the URl does not changes, it is both static and final. So you can convert this variable to 

private static final LOGIN_URL (the way, it makes a difference is, it improves readability in test that this is a fixed value and not a variable)

Reply:
changed to private static final String LOGIN_URL